### PR TITLE
Include the start of a Tips for Filtering Data guide

### PR DESCRIPTION
**Pull request by kopertop**
## Change Summary

There are many different places where `filter_by` syntax is referenced, and the only guide so far appears to be semi-hidden under a long table item in https://typesense.org/docs/27.0/api/search.html#filter-parameters. That's less than ideal for how to view the full details of the `filter_by` syntax, and that doesn't include information like filtering based on Joins (which are new).

This is the start of an attempt to reconcile the different search filtering options, and provide a better guide for formatting those filter strings.

## PR Checklist

- [x] I have read and signed the [Contributor License Agreement](https://forms.gle/PZyiY5N2GDQU8GsV9).

### CODE REVIEW

File: docs-site/content/guide/tips-for-filtering.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+If you want to filter for a value that includes special characters, such as a comma (`,`), you can use
+backticks around your filter value:
+
+```
+country:=`United States, Minor Outlying Islands`
+```
+
+## Filtering Geopoints
+
+For information on filtering `geopoint` fields to search within a specific area, please refer to our <RouterLink :to="`/${$site.themeConfig.typesenseLatestVersion}/api/geosearch.html#searching-within-a-radius`">documentation</RouterLink> on Geosearch.
```
Comment (jasonbosco) [inline docs-site/content/guide/tips-for-filtering.md:204]: @tharropoulos Could you add actual examples here of geo filtering, just like in the JOINs section, and then link to the geo search docs?

**Review by jasonbosco — CHANGES_REQUESTED** [review]
This is awesome! Thank you @kopertop and @tharropoulos! 

@tharropoulos - I've made a few tweaks and I've added one comment for you. Once that's addressed, we can merge this in.

File: docs-site/content/guide/tips-for-filtering.md (hunk 1 above)
Comment (tharropoulos) [reply docs-site/content/guide/tips-for-filtering.md:221]: > @tharropoulos Could you add actual examples here of geo filtering, just like in the JOINs section, and then link to the geo search docs?

Addressed in 29f67a3ac78f3e7a3b18b54029c6d299fac9de4f

**Review by jasonbosco — APPROVED** [review]

**Comment by jasonbosco** [comment]
Looks great!

**Pull request merged**